### DISCUSSION TITLE
Crazyradio 2 connection issue on just one of my three computers

### DISCUSSION THREAD

**Discussion by simondlevy** [bitcraze/discussions#772] [Q&A] [answered]
By following the instructions of @knmcguire I've been able to get my Crazyradio 2.0 talking to my Crazyflie 2.1 on both a Windows and Linux workstation; however, for some reason I can't get the client to detect the radio on my Windows laptop (Acer Predator / Windows 10 / Python 3.7). It's a mystery, because I used Zadig to install the driver and can see the Crazyradio 2.0 in Device Manager, but no luck getting the client to talk to it (WARNING:cfclient.ui.main:Cannot find a Crazyradio Dongle).

Advice appreciated!

**Comment by ataffanel** [comment]
What driver did you install with Zadig? The client is not compatible with `winusb` so you need to make sure you have installed the `libusb` driver.

**Reply by simondlevy** [reply]
Thanks for the suggestion, Arnaud.  I did have the winusb driver installed (which worked fine on my WIndows workstation), so I installed the libusb driver on my laptop but still had the same error.  Here's the client's output up to and including the "Cannot find Crazyradio Dongle" message:

UserWarning: Using SDL2 binaries from pysdl2-dll 2.24.0
INFO:cfclient.gui:Disabling STL printouts
INFO:cfclient.utils.input.inputreaders:Input readers: ['linuxjsdev', 'pysdl2']
INFO:cfclient.utils.input.inputreaders:Could not initialize [linuxjsdev]: Only supported on Linux
INFO:cfclient.utils.input.inputreaders:Successfully initialized [pysdl2]
INFO:cfclient.utils.input.inputinterfaces:Found interfaces: ['leapmotion', 'wiimote', 'zmqpull']
INFO:cfclient.utils.input.inputinterfaces:Could not initialize [leapmotion]: Leap Motion library probably not installed (No module named 'leapsdk')
INFO:cfclient.utils.input.inputinterfaces:Could not initialize [wiimote]: Missing cwiid (wiimote) driver No module named 'cwiid'
INFO:cfclient.utils.config:Dist config read from C:\Users\levys\AppData\Local\Programs\Python\Python37\lib\site-packages\cfclient/configs/config.json
INFO:cfclient.utils.config:Config file read from [C:\Users\levys\AppData\Local\Bitcraze\cfclient/config.json]
INFO:cfclient.utils.input.inputinterfaces:Could not initialize [zmqpull]: ZMQ input disabled in config file
INFO:cfclient.utils.zmq_led_driver:ZMQ led disabled in config file
INFO:cfclient.utils.zmq_param:ZMQ param disabled in config file
INFO:cfclient.utils.input:Using device blacklist [(VirtualBox|VMware)]
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cflib.drivers.cfusb:Looking for devices....
INFO:cflib.crtp.prrtdriver:Initialized PRRT driver.
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cfclient.ui.tab_toolbox:No config found for toolbox_areas
INFO:cfclient.ui.tab_toolbox:No config found for open_toolboxes
INFO:cfclient.ui.tab_toolbox:No config found for open_toolboxes
INFO:cfclient.ui.tabs.QualisysTab:Exception during qtm discovery: 
INFO:cfclient.utils.input.inputreaders.pysdl2:Looking for devices
INFO:cfclient.utils.input.inputreaders.pysdl2:Found 1 devices
INFO:cfclient.utils.input:Selected MUX: Normal
INFO:cfclient.ui.main:Role of 2In1 USB Joystick is Device
INFO:cfclient.utils.input.mux:Adding device 2In1 USB Joystick to MUX Normal
WARNING:cfclient.ui.main:Cannot find a Crazyradio Dongle
INFO:cflib.drivers.cfusb:Looking for devices....
INFO:cflib.crtp.prrtdriver:Initialized PRRT driver.
INFO:cfclient.utils.config:Config file saved to [C:\Users\levys\AppData\Local\Bitcraze\cfclient/config.json]
INFO:cflib.crazyflie:Closing link
INFO:cflib.crazyflie:Callback->Disconnected from []
INFO:cfclient.ui.tabs.QualisysTab:Crazyflie disconnected from 
Cannot find a Crazyradio Dongle
Cannot find a Crazyradio Dongle

**Comment by simondlevy** [comment, marked as answer]
Scratch that last answer -- I tried once more, and it worked!  I must have installed the libusb driver for the wrong device.  Thank you again for the tip.
